### PR TITLE
[otbn] Pass the start address to the ISS

### PR DESCRIPTION
**Pull request by imphil**
The start address is necessary to execute any real-world algorithm, pass
it on to the ISS.

This will need #3312 to go in first.

### CODE REVIEW

File: hw/ip/otbn/dv/model/otbn_model.cc [hunk 1]
```diff
@@ -262,7 +264,7 @@ extern "C" int run_model(const char *imem_scope, int imem_words,
 
   std::ostringstream cmd;
   cmd << model_path << " " << imem_words << " " << ifname << " " << dmem_words
-      << " " << dfname << " " << cfname << " " << tfname;
+      << " " << dfname << " " << cfname << " " << tfname << " " << start_addr;
```
Comment (rswarbrick) [inline hw/ip/otbn/dv/model/otbn_model.cc:267]: I feel like this command line is getting a little crazy. Would you be happy if we dumped a configuration file (JSON?) or something and read it in the Python? Obviously, not part of this PR, but it would be good to get your thoughts.

**Review by rswarbrick — APPROVED** [review]
Looks good to me! Also, assuming you tested this, it's nice to know it works! I don't think I actually tried fiddling with `reset_addr` when I was hacking on the simulator :-)

File: hw/ip/otbn/dv/model/otbn_model.cc (hunk 1 above)
Comment (imphil) [reply hw/ip/otbn/dv/model/otbn_model.cc:267]: I quite enjoy the ability to run a tool without constructing a config file first. But there are still ways to make this easier to use, e.g. named arguments, or environment variables (the standard way of passing data between processes in qmail/daemontools times, I probably spent too much time with them ...)

Comment (imphil) [reply hw/ip/otbn/dv/model/otbn_model.cc:267]: I quite enjoy the ability to run a tool without constructing a config file first. But there are still ways to make this easier to use, e.g. named arguments, or environment variables (the standard way of passing data between processes in qmail/daemontools times, I probably spent too much time with them ...)

**Comment by imphil** [comment]
Rebased on top of #3312 to change the argument name to start_addr.

File: hw/ip/otbn/dv/model/otbn_model.cc (hunk 1 above)
Comment (rswarbrick) [reply hw/ip/otbn/dv/model/otbn_model.cc:267]: Yeah, or maybe a default naming scheme would be the right thing. We could pass the name of a directory containing all the stuff, and then it would look at `path/imem`, `path/dmem` etc.
